src/datasets/info.py

```diff
@@ -93,23 +93,24 @@ class DatasetInfo:
     Note: Not all fields are known on construction and may be updated later.
 
     Attributes:
-        description (str):
-        citation (str):
-        homepage (str):
-        license (str):
-        features (Features, optional):
+        description (str): A description of the dataset.
+        citation (str): A BibTeX citation of the dataset.
+        homepage (str): A URL to the official homepage for the dataset.
+        license (str): The dataset's license.
```

Comment:
```suggestion
        license (str): The dataset's license. It can be the name of the license or a paragraph containing the terms of the license.
```